packages/expo-yarn-workspaces/webpack.js

```diff
@@ -28,7 +28,7 @@ exports.createWebpackConfigAsync = async function createWebpackConfigAsync(env,
 
     // discover workspace package directories via glob - source yarn:
     // https://github.com/yarnpkg/yarn/blob/a4708b29ac74df97bac45365cba4f1d62537ceb7/src/config.js#L812-L826
-    const patterns = workspacePackage.workspaces ?? [];
+    const patterns = (workspacePackage.workspaces || workspacePackage.workspaces.packages) ?? [];
```

Comment:
If workspaces.packages is defined, then workspaces itself is defined

Reply:
Hey @ide thanks for taking a look! Sorry was OOO. I understand what you're saying but I'm not sure what you'd like for me to change.

This is what I'm trying to accomplish:
- workspacePackage.workspaces is a valid option
- workspacePackage.workspaces.packages is also a valid option
- if neither option exists, fallback to an empty array

Reply:
My understanding is you want to support both:
```json5
{ "workspaces: { "packages": [...] } }
```
and
```json5
{ "workspaces": [...] }
```

If a package uses the former syntax (`{ "workspaces: { "packages": [...] } }`), then:
```js
 const patterns = (workspacePackage.workspaces || workspacePackage.workspaces.packages) ?? [];
```
will evaluate to `patterns = { packages: [...] }` instead of `patterns = [...]`.

Reply:
Ohhh, duh 🙈 Thanks for walking me through that!